workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```
@@ -177,13 +177,22 @@ public int Run()
 
                     foreach (var outputProcessor in outputProcessors)
                     {
-                        outputProcessor(outputString);
+                        try
+                        {
+                            outputProcessor?.Invoke(outputString);
```

If the `Action` is null - we should catch that when you add the processor rather than doing `action?.Invoke()`

---

Which will silently pass over

---

I'll remove the null check and just let it fail, the exception will explain nicely.